components/root/root.jsx

```diff
@@ -307,6 +312,10 @@ export default class Root extends React.Component {
                         path={'/help'}
                         component={HelpController}
                     />
+                    <LoggedInHFTRoute
+                        path={'/terms_of_service'}
```

Comment:
nit: throughout the server code, we're using the term `service terms`. I thought this was a bit odd, since I'm used to `terms of service`, but at least it was consistent. Should we converge on `service terms` here? Or `terms of service` everywhere?